modules/nf-scil/segmentation/fastseg/main.nf

```diff
@@ -0,0 +1,71 @@
+
+process SEGMENTATION_FASTSEG {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+        tuple val(meta), path(image)
+
+    output:
+        tuple val(meta), path("*mask_wm.nii.gz")                , emit: wm_mask
+        tuple val(meta), path("*mask_gm.nii.gz")                , emit: gm_mask
+        tuple val(meta), path("*mask_csf.nii.gz")               , emit: csf_mask
+        tuple val(meta), path("*map_wm.nii.gz")                 , emit: wm_map
+        tuple val(meta), path("*map_gm.nii.gz")                 , emit: gm_map
+        tuple val(meta), path("*map_csf.nii.gz")                , emit: csf_map
+        path "versions.yml"                                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def number_of_tissues = task.ext.number_of_tissues ? "-n " + task.ext.number_of_tissues : ""
+
+    """
+    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
+    export OMP_NUM_THREADS=1
+    export OPENBLAS_NUM_THREADS=1
+
+    fast -t 1 $number_of_tissues\
```

Comment:
I'm not a user of `fast`, I have a question about the behavior. Is there a reason to raise the number of tissues over 3 ? Because the rest of the script awaits à 3-tissues segmentation (or at least on with 0/1/2 ordered as csf/gm/wm). If not, remove the parameter and use `3` directly in the script.

Reply:
I think in our use case there isn't really a reason to do so, I fixed it to 3 directly in the script.